SixTrack/sixtrack.s

```diff
@@ -29578,7 +29665,11 @@ C     Convert r(1), r(2) from U(0,1) -> rvec0 as Gaussian with cutoff mcut (#sig
 !
       do i = 1, numeff
 !hr08   rsig(i) = dble(i)/2d0 - 0.5d0 + 6d0
-        rsig(i) = (dble(i)/2d0 - 0.5d0) + 6d0                            !hr08
+        rsig(i) = (dble(i)/2d0 - 0.5d0) + 5d0                           !hr08
```

Comment:
Yes, actually I have extended the range of the histogram, there is no impact on other people.